classes/Device.class.php

```diff
@@ -1871,7 +1871,7 @@ function GetChildDevicePicture($parentDetails, $rear=false){
 						$slot->H=$ymax-$ymin;
 					}else{
 						// Last slot isn't defined so just error out
-						break;
+						return;
```

Comment:
Was this change fixing something specific?  There are some oddities in that function that led to things that look weird.